CRM/Core/BAO/CustomField.php

```diff
@@ -2544,7 +2551,10 @@ public static function isSerialized($field) {
     // Fields retrieved via api are an array, or from the dao are an object. We'll accept either.
     $field = (array) $field;
     // FIXME: Currently the only way to know if data is serialized is by looking at the html_type. It would be cleaner to decouple this.
-    return ($field['html_type'] == 'CheckBox' || strpos($field['html_type'], 'Multi') !== FALSE);
+    return ($field['html_type'] == 'CheckBox'
+      || ($field['html_type'] == 'Autocomplete-Select' && $field['data_type'] != 'ContactReference')
+      || strpos($field['html_type'], 'Multi') !== FALSE
```

Comment:
This logic is incorrect. According to your screenshot, the fact that html_type == Autocomplete-Select is insufficient to know if it's multivalued, you also have to check the attributes.

Reply:
Hi @colemanw 

The code already handles for single value Autocomplete-Select if its serialized. I thought of serializing every auto-complete field. Should i restrict it to only if its multi-value?

Thanks
Pradeep

Reply:
Yes, then we can accurately check the serialize metadata for the field.